key.go

```diff
@@ -398,6 +398,18 @@ func (k *Key) InUint64(defaultVal uint64, candidates []uint64) uint64 {
 	return defaultVal
 }
 
+// InBool always returns value without error,
+// it returns default value if error occurs or doesn't fit into candidates.
+func (k *Key) InBool(defaultVal bool, candidates []bool) bool {
```

Comment:
Thanks for adding these!

But this method seems particularly not useful in practice because boolean only have two values: true and false. What do you think?

Reply:
Yeah, I agree, removing.